EXP-Client/todo/AddTaskMenu.js

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { Provider } from 'react-native-paper';
+import {
+  StyleSheet,
+  Button,
+  Text,
+  SafeAreaView,
+  View
+} from 'react-native';
+import { TextInput, DatePicker } from 'react-native-paper';
+import { BottomSheet } from 'react-native-btr';
+import { Rating } from 'react-native-ratings';
+import { en, registerTranslation, DatePickerInput } from 'react-native-paper-dates';
+
+// for date picker
+registerTranslation('en', en);
+
+export default function AddTaskMenu({ isPanelActive, setIsPanelActive }) {
+  const [name, setName] = React.useState('');
+  const [deadline, setDeadline] = React.useState();
+  const [rating, setRating] = React.useState(0);
+
+  const handleAddTask = () => {
+    setIsPanelActive(false);
+    const newTask = {
+      taskName: name,
+      deadline: deadline,
+      priority: rating,
+      completed: false
+    };
+    console.log(newTask)
+
+    let instance = axios.create({
+      baseURL: "https://explife-backend.fly.dev"
+    });
+
+    instance
+      .post("/lists/:id", newDonut)
+      .then(() => {
+        // Clear inputs
+        setName('')
+        setDeadline(new Date())
+        setRating(0)
+
+        // Refresh list
+        getTasks()
+      })
+      .catch((error) => {
+        console.log(error)
```

Comment:
Less of a priority but we might want to have more involved error checking later on

Reply:
Middleware code isn't implemented yet